tests/document/indexes.py

```diff
@@ -792,6 +792,27 @@ class User(Document):
 
         User.drop_collection()
 
+    def test_unique_and_primary_create(self):
+        """Create a new record with a duplicate primary key
+        throws an exception
+        """
+
```

Comment:
That's a good question @ephraimberkovitch. It's basically a tradeoff between PR isolation (where we want to touch *solely* the parts of the code that are logically related to the particular problem we're solving) vs passive cleanup. Personally I like applying cosmetic tweaks to unrelated code as I go as long as it doesn't make the PR considerably less readable. Otherwise the likelihood of going back and tweaking it in a separate PR are quite low.

Your PR in its current form is still readable and the cleanup you did is very welcome!